docker/base/Dockerfile

```diff
@@ -0,0 +1,110 @@
+ARG BASE_IMAGE=nvcr.io/nvidia/l4t-pytorch:r32.4.3-pth1.6-py3
+FROM ${BASE_IMAGE}
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# ====================
+# INSTALL COMMON TOOLS
+# ====================
+RUN apt-get update && apt-get install -y net-tools vim cmake
+
+
```

Comment:
Do you want to add pre-caching of torchvision models?

```
#
# pre-cache torchvision models
#
RUN python3 -c "import torchvision; \
                model = torchvision.models.alexnet(pretrained=True); \
                model = torchvision.models.squeezenet1_1(pretrained=True); \
                model = torchvision.models.resnet18(pretrained=True); \
                model = torchvision.models.resnet34(pretrained=True)"
```

Reply:
Yes these seems good.

I've added this as dependency of Jupyter container.